ush/python/pygfs/task/aero_bmatrix.py

```diff
@@ -40,6 +40,9 @@ def __init__(self, config):
         _res_anl = int(self.task_config['CASE_ANL'][1:])
         _window_begin = add_to_datetime(self.task_config.current_cycle, -to_timedelta(f"{self.task_config['assim_freq']}H") / 2)
 
+        # fix ocnres
+        self.task_config.OCNRES = f"{self.task_config.OCNRES:03d}"
```

Comment:
aah yes, `025` gets turned into an int `25`.
Sometime we should probably turn `OCNRES` into `mx025`, just like `CASE` has `C` in `C768`.

Reply:
yeah, this comes up in a lot of places (marine and snow DA already also), it would be better to fix it once and for all